.github/workflows/main-build.yml

```diff
@@ -73,7 +84,7 @@ jobs:
       - uses: actions/setup-python@v2
```

Comment:
I think we may actually be able to remove setup-python - unlike the other languages we build in docker so python shouldn't be used. Either way, we don't need this setup python to be parameterized to 3.9 / 3.8, we just need the runtime set below to check both